scripts/safe_template_linter.py

```diff
@@ -777,12 +778,13 @@ def _find_unescaped_expressions(self, underscore_template):
 
         expressions = []
         for match in unescaped_expression_regex.finditer(underscore_template):
-            expression = {
-                'start_index': match.start(),
-                'end_index': match.end(),
-                'expression': match.group(),
-            }
-            expressions.append(expression)
+            if "HtmlUtils" not in match.group():
```

Comment:
Thanks for getting this started.  There are tests, but I don't think it is used for Underscore.js yet.  I want to see if we can make this have fewer false positives, but this is closer to what we want.